src/scheduler/animationFrame.ts

```diff
@@ -1,4 +1,32 @@
 import { AnimationFrameAction } from './AnimationFrameAction';
 import { AnimationFrameScheduler } from './AnimationFrameScheduler';
 
+/**
+ *
+ * Animation Frame Scheduler
+ *
+ * <span class="informal">Perform task when `window.requestAnimationFrame` would fire</span>
+ *
+ * When `animationFrame` scheduler is used with delay, it will fall back to {@link async} scheduler
+ * behaviour.
+ *
+ * Without delay, `animationFrame` scheduler can be used to create smooth browser animations.
+ * It makes sure scheduled task will happen just before next browser content repaint,
+ * thus performing animations as efficiently as possible.
+ *
+ * @example <caption>Schedule div height animation</caption>
+ * const div = document.querySelector('#some-div');
+ *
+ * Rx.Scheduler.schedule(function(height) {
+ *   div.style.height = height + "px"
+ *
```

Comment:
Perhaps add a comment here explaining that `this` is a reference to the scheduled `Action`?